.github/workflows/tests.yaml

```diff
@@ -22,11 +22,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: dev env setup
         uses: ./.github/actions/dev-env-setup
-      - name: run app locally
-        uses: ./.github/actions/local-app-run
       - name: Run Jest tests
-        working-directory: ./app
-        run: yarn test
+        uses: ./.github/actions/jest-test
```

Comment:
We're running jest tests twice in our CI. Can't we just have the sonarcloud step here?

Reply:
Good point, yeah I can go ahead and move the sonarcloud run in the tests workflow